tests/test_ops/test_ops.py

```diff
@@ -765,11 +767,11 @@ def test_gather(backend,
     # so the ncnn_outputs has 2 dimensions, not 1.
     import importlib
 
-    import onnxruntime
-    assert importlib.util.find_spec('onnxruntime') is not None, \
-        'onnxruntime not installed.'
+    assert importlib.util.find_spec('onnxruntime') is not None, 'onnxruntime \
```

Comment:
The test should be skipped instead of throw assert error.
And you could create a constant input/output instead using onnxruntime as the benchmark